.github/workflows/perl-openwsman.yml

```
@@ -164,11 +166,13 @@ jobs:
           distrib: ${{ matrix.distrib }}
           package_extension: ${{ matrix.package_extension }}
           arch: ${{ matrix.arch }}
+          release: 2
```

perl-openwsman last release was `3`
```suggestion
          release: 4
```